assets/src/edit-story/panels/font.js

```diff
@@ -26,27 +28,40 @@ function FontPanel( { selectedElements, onSetProperties } ) {
 		onSetProperties( state );
 		evt.preventDefault();
 	};
+
+	const fontWeights = [
```

Comment:
Later, we should be move this settings somewhere else.

Reply:
Wouldn't these (ideally) be on a per-font basis and come from the fonts array? Similar for styles?

Reply:
This is now fixed.